### DISCUSSION TITLE
Unrecognized Conversion Type Error

### DISCUSSION THREAD

**Discussion by ZachWatsonSC** [grafana/tanka#815] [Q&A]
I'm following the deployment steps [here](https://grafana.com/docs/loki/latest/installation/tanka/) and trying to run the final command of `tk show environments/loki`, but when running that command I only get the following error:
`Error: evaluating jsonnet: RUNTIME ERROR: Unrecognized conversion type: ;`
This error originates in the `vendor/loki/gateway.libsonnet` file in the `configMap.withData` call. I haven't been able to find any open or closed issues, existing discussion posts, or anything online about this, so I'm not sure if this is just a config issue or if there's something wrong with the installation. Any help would be appreciated. Thanks!

**Comment by Duologic** [comment]
Can you share your main.jsonnet file?

**Comment by ZachWatsonSC** [comment]
Hey @Duologic. It was a config error on the Loki side. Someone made a PR leaving out a formatting token in the `gateway.libsonnet` which was causing the error. I made another PR on the Loki repository to fix it, so this isn't an issue with Tanka itself. I meant to update this post but I guess I forgot to.

**Reply by Duologic** [reply]
Perhaps link the PRs for reference?

**Reply by ZachWatsonSC** [reply]
[My PR](https://github.com/grafana/loki/pull/8580)
The PR that I'm fixing is linked in my PR
